Source/AwsCommonRuntimeKit/mqtt/Mqtt5Types.swift

```diff
@@ -1029,7 +1038,7 @@ public class MqttClientOptions: CStruct {
         }
 
     typealias RawType = aws_mqtt5_client_options
-    func withCStruct<Result>( _ body: (aws_mqtt5_client_options) -> Result) -> Result {
+    func withCStruct<Result>(userData: UnsafeMutableRawPointer?, _ body: (aws_mqtt5_client_options) -> Result) -> Result {
```

Comment:
While this `userData` will be passed down as individual callback's `userData` property, at this point it will only ever be the callbackCore. It should NOT be anything other than the callbackCore pointer. Should we rename this to be more representative of what is expected at this point?

Reply:
The function is override the base function for all CStruct requires userData. We could not rename the parameter. I can definitely add more comments to indicate the userData here should be the callbackCore.